addons/account/models/account.py

```diff
@@ -416,10 +416,21 @@ def _default_outbound_payment_methods(self):
     bank_acc_number = fields.Char(related='bank_account_id.acc_number')
     bank_id = fields.Many2one('res.bank', related='bank_account_id.bank_id')
 
+    # alias configuration for 'purchase' type journals
+    alias_id = fields.Many2one('mail.alias', string='Alias')
```

Comment:
Why don't you use the mail.alias.mixin class ?

Reply:
We wanted user to be able to set alias for `purchase` type journals only and if we use `mail.alias.mixin` then it would create an empty alias for each journal regardless of it's type, that is, an empty alias will be created for journals of type sale/bank/cash/general which we will never going to use. Do you think we should still use `mail.alias.mixin` ?

Reply:
@qdp-odoo now i'm reviewing https://github.com/odoo/odoo/pull/24709 and I'm very sad that tde's comment wasn't taken into account. Would it be possible to fix that and use mail.alias.mixin ?